docs/user/customization.md

```diff
@@ -60,9 +59,15 @@ The following `install-config.yaml` properties are available:
     * `httpProxy` (optional string): The URL of the proxy for HTTP requests.
     * `httpsProxy` (optional string): The URL of the proxy for HTTPS requests.
     * `noProxy` (optional string): A comma-separated list of domains and [CIDRs][cidr-notation] for which the proxy should not be used.
+* `publish` (optional string): This controls how the user facing endpoints of the cluster like the Kubernetes API, OpenShift routes etc. are exposed.
+    Valid values are `External` (the default) and `Internal`.
 * `pullSecret` (required string): The secret to use when pulling images.
 * `sshKey` (optional string): The public Secure Shell (SSH) key to provide access to instances.
 
+### Capabilities
+
+* `baselineCapabilitySet` (optional string): Selects an initial set of optional capabilities to enable. Valid values are `vCurrent` (the default), `v4.11` and `None`.
+* `additionalEnabledCapabilities` (optional array of strings): Extends the set of managed capabilities beyond the baseline defined in `baselineCapabilitySet`. Default is an empty set. Valid values are `openshift-samples` and `baremetal`.
```

Comment:
You validate these, right?  Have you bumped your vendored openshift/api to pull in the `baremetal` addition?

Reply:
This line will need a bump for `marketplace` for #5755.  If #5755 lands first, then this PR would need the bump.  If this PR lands first, then #5755 would need the bump.  Or a third follow up PR could come in with the bump behind both this PR and #5755.

Reply:
> This line will need a bump for `marketplace` for #5755. If #5755 lands first, then this PR would need the bump. If this PR lands first, then #5755 would need the bump. Or a third follow up PR could come in with the bump behind both this PR and #5755.

My guess is probably option 3. I'll send a follow up PR with new vendoring and an update to this doc, which should be the norm anyway going forward. Thanks.